dmoj/cptbox/sandbox.py

```diff
@@ -214,15 +221,21 @@ def r_execution_time(self):
 
     def kill(self):
         log.warning('Request the killing of process: %s', self.pid)
-        print('Child is requested to be killed', file=sys.stderr)
         try:
             os.killpg(self.pid, signal.SIGKILL)
         except OSError:
             import traceback
             traceback.print_exc()
 
     def _callback(self, syscall):
-        callback = self._callbacks[syscall]
+        try:
+            callback = self._callbacks[syscall]
+        except IndexError:
+            if self._syscall_index == 3:
```

Comment:
Do the ARM-specific syscalls not exist on ARM64?

Reply:
Definitely not. The ARM system call table is not the same as the ARM64 one. The ARM64 is the new unified table with the minimum number of system calls.